modules/apps/portal-settings/portal-settings-web/src/main/resources/META-INF/resources/user_layout_configuration.jsp

```diff
@@ -0,0 +1,75 @@
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<aui:input name="<%= Constants.CMD %>" type="hidden" value="<%= Constants.UPDATE %>" />
+
+<div class="row">
+	<div class="col-md-12">
+		<aui:input checked="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LAYOUT_USER_PRIVATE_LAYOUTS_ENABLED) %>" id="enablePrivateLayouts" inlineLabel="right" label="enable-private-layouts" labelCssClass="simple-toggle-switch" name='<%= "settings--" + PropsKeys.LAYOUT_USER_PRIVATE_LAYOUTS_ENABLED + "--" %>' onClick="togglePrivateLayoutsAutoCreate()" type="toggle-switch" value="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LAYOUT_USER_PRIVATE_LAYOUTS_ENABLED) %>" />
+
+		<aui:input checked="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LAYOUT_USER_PRIVATE_LAYOUTS_AUTO_CREATE) %>" id="enablePrivateLayoutsAutoCreate" inlineLabel="right" label="enable-private-layouts-auto-create" labelCssClass="simple-toggle-switch" name='<%= "settings--" + PropsKeys.LAYOUT_USER_PRIVATE_LAYOUTS_AUTO_CREATE + "--" %>' type="toggle-switch" value="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LAYOUT_USER_PRIVATE_LAYOUTS_AUTO_CREATE) %>" />
+
+		<aui:input checked="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LAYOUT_USER_PUBLIC_LAYOUTS_ENABLED) %>" id="enablePublicLayouts" inlineLabel="right" label="enable-public-layouts" labelCssClass="simple-toggle-switch" name='<%= "settings--" + PropsKeys.LAYOUT_USER_PUBLIC_LAYOUTS_ENABLED + "--" %>' onClick="togglePublicLayoutsAutoCreate()" type="toggle-switch" value="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LAYOUT_USER_PUBLIC_LAYOUTS_ENABLED) %>" />
+
+		<aui:input checked="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LAYOUT_USER_PUBLIC_LAYOUTS_AUTO_CREATE) %>" id="enablePublicLayoutsAutoCreate" inlineLabel="right" label="enable-public-layouts-auto-create" labelCssClass="simple-toggle-switch" name='<%= "settings--" + PropsKeys.LAYOUT_USER_PUBLIC_LAYOUTS_AUTO_CREATE + "--" %>' type="toggle-switch" value="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LAYOUT_USER_PUBLIC_LAYOUTS_AUTO_CREATE) %>" />
+	</div>
+</div>
+
+<script>
+	let enablePrivateLayoutsElement = document.getElementById(
+		'<portlet:namespace />enablePrivateLayouts'
+	);
+
+	let enablePrivateLayoutsAutoCreateElement = document.getElementById(
+		'<portlet:namespace />enablePrivateLayoutsAutoCreate'
+	);
+
+	if (enablePrivateLayoutsElement.checked === false) {
+		enablePrivateLayoutsAutoCreateElement.setAttribute('disabled', '');
+	}
+
+	let enablePublicLayoutsElement = document.getElementById(
+		'<portlet:namespace />enablePublicLayouts'
+	);
+	let enablePublicLayoutsAutoCreateElement = document.getElementById(
+		'<portlet:namespace />enablePublicLayoutsAutoCreate'
+	);
+
+	if (enablePublicLayoutsElement.checked === false) {
+		enablePublicLayoutsAutoCreateElement.setAttribute('disabled', '');
+	}
```

Comment:
@torresvictor100, please use the operator `!` instead due to follow the code pattern used in the JavaScript funcions below